core/app/models/concerns/spree/product_scopes.rb

```diff
@@ -43,23 +43,23 @@ def self.property_conditions(property)
       add_simple_scopes simple_scopes
 
       add_search_scope :ascend_by_master_price do
-        joins(master: :default_price).order("#{price_table_name}.amount ASC")
```

Comment:
Why this is not needed anymore?

Reply:
@mad-eel now it's defined here: https://github.com/spree/spree/pull/11019/files#diff-56955ab06005de319149d735d43fe73556d500e98cb0063492985c1165848c91R203

Before it only took the default price into consideration, meaning it would skip any non-default currency prices